src/Services/InsuranceCompanyService.php

```diff
@@ -110,16 +167,10 @@ public function getInsuranceTypes()
         );
     }
 
-    public function getFreshId()
-    {
-        $id = sqlQuery("SELECT MAX(id)+1 AS id FROM insurance_companies");
-
-        return $id['id'];
-    }
 
     public function insert($data)
     {
-        $freshId = $this->getFreshId();
+        $freshId = $this->getFreshId("id", "insurance_companies");
```

Comment:
Did you move getFreshId() somewhere? Looks like you meant to add to base service or ?

Reply:
BaseService already has getFreshId($idField, $table), hence I removed the function.